packages/react-native-client/src/hooks/useScreenShare.ts

```diff
@@ -108,6 +91,29 @@ export function useScreenShare() {
     return 'denied';
   }, []);
 
+  const toggleScreenShare = useCallback(
+    async (screenShareOptions: Partial<ScreenShareOptions> = {}) => {
+      if (Platform.OS === 'android' && !isScreenShareOn) {
+        (await handleScreenSharePermission()) == 'granted' &&
+          (await RNFishjamClientModule.startForegroundService({
+            enableScreenSharing: true,
+          }));
+      }
+      const options = {
+        ...screenShareOptions,
+        screenShareMetadata: {
+          displayName: 'presenting',
+          type: 'screensharing' as const,
+          active: !isScreenShareOn,
+        },
+      };
+      await RNFishjamClientModule.toggleScreenShare(options);
+      screenShareSimulcastConfig = defaultSimulcastConfig(); //to do: sync with camera settings
```

Comment:
**suggestion:** 
What that comment is about?

Reply:
I have no idea, it shows as changed because of refactoring. 

Maybe @mironiasty remembers? 
https://github.com/fishjam-cloud/mobile-client-sdk/pull/84/files